In [47]:
from keras import models
import os
from keras.preprocessing.image import ImageDataGenerator
import scipy

In [9]:
import pandas
import numpy
import matplotlib.pyplot as plt

In [10]:
from keras.applications import MobileNet

In [11]:
from keras.layers import activations,Dropout,Flatten,Conv2D,Dense

# Importing the dataset


In [60]:
PATH = os.getcwd()
 
image_train_path = '/Users/shekharpalit/Desktop/github/RadAI/data/train/images/'
image_test_path = '/Users/shekharpalit/Desktop/github/RadAI/data/test/images/'
image_train_datagen = ImageDataGenerator(rescale=1./255, shear_range= 0.2, zoom_range=0.2, horizontal_flip=True)
image_test_datagen = ImageDataGenerator(rescale=1./255)
image_training_set = train_datagen.flow_from_directory(train_path, target_size = (64,64),
                                                 batch_size = 32, 
                                                 class_mode = 'binary')
image_testing_set = test_datagen.flow_from_directory(test_path, target_size = (64,64),
                                                 batch_size = 32, 
                                                 class_mode = 'binary')

mask_train_path = '/Users/shekharpalit/Desktop/github/RadAI/data/train/mask/'
mask_test_path = '/Users/shekharpalit/Desktop/github/RadAI/data/test/mask/'
mask_train_datagen = ImageDataGenerator(rescale=1./255, shear_range= 0.2, zoom_range=0.2, horizontal_flip=True)
mask_test_datagen = ImageDataGenerator(rescale=1./255)
mask_training_set = train_datagen.flow_from_directory(train_path, target_size = (64,64),
                                                 batch_size = 32, 
                                                 class_mode = 'binary')
mask_testing_set = test_datagen.flow_from_directory(test_path, target_size = (64,64),
                                                 batch_size = 32, 
                                                 class_mode = 'binary')


Found 31662 images belonging to 11 classes.
Found 10818 images belonging to 4 classes.
Found 31662 images belonging to 11 classes.
Found 10818 images belonging to 4 classes.


### Number of Classes 
- Found 31662 images belonging to 11 classes for train dataset in images
- Found 10818 images belonging to 4 classes for test dataset in images
- Found 31662 images belonging to 11 classes for train dataset in maskin(labels)
- Found 10818 images belonging to 4 classes for test dataset in masking(labels)